e_chil.c

```diff
@@ -985,8 +981,11 @@ static EVP_PKEY *hwcrhk_load_privkey(ENGINE *eng, const char *key_id,
 
```

Comment:
Looking at the lines above, I get worried...  if `res == NULL`, it does add an error on the error stack, but doesn't jump to `err`?  That looks like a major memory leak waiting to happen...

Reply:
I do check for res being NULL:
if (rtmp == NULL || res == NULL) {

Reply:
Never mind, the check I'm pointing at (just before the `err:` lable) only comes into play when RSA is disabled